project/Build.scala

```diff
@@ -0,0 +1,43 @@
+import sbt._
+import sbt.Keys._
+import sbt.internal.BuildDef
+
+import net.vonbuchholtz.sbt.dependencycheck.DependencyCheckPlugin.autoImport.dependencyCheckFailBuildOnCVSS
+
+object Build extends BuildDef with BuildCommon {
+  override def projects: Seq[Project] = mainProjects ++ testProjects
+  lazy val mainProjects: Seq[Project] = schemaVersions.map(nitfProject)
+  lazy val testProjects: Seq[Project] = schemaVersions.map(testProject)
+
+  private val releaseVersion = 0
```

Comment:
Do we have to update that value on each new release? Is there a way to move that out into a `version.sbt` file?

Reply:
I was thinking that we'll get rid of this by adding the `sbt-release` plugin and configuring it to manage this automatically. I'm _guessing_ that the plugin would generate `version.sbt` automatically. Is this correct?

Reply:
Yes indeed